packages/block-editor/src/components/index.js

```diff
@@ -118,6 +118,7 @@ export {
 export { default as Warning } from './warning';
 export { default as WritingFlow } from './writing-flow';
 export { useCanvasClickRedirect as __unstableUseCanvasClickRedirect } from './use-canvas-click-redirect';
+export { default as __experimentalUseBlockDisplayInformation } from './use-block-display-information';
```

Comment:
Any reason why it should be experimental? I personally can't think of anything missing so it cab be considered final.

Reply:
Now that happened, [CHANGELOG](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/CHANGELOG.md) could benefit from a note as well. It doesn't feel like it's updated frequently :(